Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -113,8 +114,10 @@ public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string
 			data.AddPropertyIfValue("settings", settings);
 			data.AddPropertyIfValue("tracking_fields", trackingFields?.Select(tf => new JObject() { { "field", tf.Key }, { "value", tf.Value } }));
 
+			string templateIdUrl = string.IsNullOrWhiteSpace(templateId) ? string.Empty : $"?template_id={templateId}";
```

Comment:
This is unnecessary, please remove this line and see my next comment for a simpler way to append querystring parameters to a request.

Reply:
Code line removed.